scene/gui/graph_edit.h

```diff
@@ -126,6 +126,11 @@ class GraphEdit : public Control {
 		SCROLL_PANS,
 	};
 
+	enum GridPattern {
+		LINES,
+		DOTS
```

Comment:
```suggestion
		GRID_PATTERN_LINES,
		GRID_PATTERN_DOTS
```
Name `enums` descriptively